code/modules/mob/living/simple_animal/aliens/drone.dm

```diff
@@ -1,39 +1,32 @@
 
 //malfunctioning combat drones
-/mob/living/simple_animal/hostile/retaliate/malf_drone
+/mob/living/simple_animal/hostile/malf_drone
 	name = "combat drone"
 	desc = "An automated combat drone armed with state of the art weaponry and shielding."
 	icon_state = "drone3"
 	icon_living = "drone3"
 	icon_dead = "drone_dead"
-	ranged = 1
-	rapid = 1
-	speak_chance = 5
+
+	faction = "malf_drone"
```

Comment:
I may be wrong, but I think this was specifically **not** a thing, to show that the drones are really broken. Don't personally mind if this changes, but it might bear discussion.